app/Policies/LicensePolicy.php

```diff
@@ -42,8 +42,12 @@ public function viewKeys(User $user, License $license = null)
      * @param  \App\Models\User  $user
      * @return mixed
      */
-    public function files(User $user)
+    public function files(User $user, $license = null)
     {
-        return $user->hasAccess($this->columnName().'.files');
+        if ($user->hasAccess('licenses.files'))  {
```

Comment:
This *looks* like it probably ought to be the same as the more generic 'files' policy that's defined in SnipePermissionsPolicy? Though the column name might not be 'licenses' (it may actually be `licenseseats` instead maybe?).

If that's the case I would explicitly mention that in a comment. If not, I think you can remove this method and just use the inherited one? (Unless you're planning on beefing this up in a subsequent release).

Reply:
I am planning on beefing this up - and potentially reworking a lot of our gates - in the future :)